public/js/components/RightSidebar.js

```diff
@@ -88,15 +88,15 @@ const RightSidebar = React.createClass({
     keyShortcuts.on("F8", this.resume);
     keyShortcuts.on("F10", this.stepOver);
     keyShortcuts.on("F11", this.stepIn);
-    keyShortcuts.on("F12", this.stepOut);
+    keyShortcuts.on("Shift+F11", this.stepOut);
   },
 
   componentWillUnmount() {
     const { keyShortcuts } = this.props;
     keyShortcuts.off("F8", this.resume);
     keyShortcuts.off("F10", this.stepOver);
     keyShortcuts.off("F11", this.stepIn);
-    keyShortcuts.off("F12", this.stepOut);
+    keyShortcuts.off("Shift+F11", this.stepOut);
```

Comment:
Yes. The old debugger has `Shift+F11` for `Step Out`. I wrote the code for the keyboard shortcuts.. and I think I chose `F12` by mistake. I will rectify that here.